packages/node-sdk/src/client.ts

```diff
@@ -234,9 +234,15 @@ export class BucketClient {
         TBody,
         { success: boolean }
       >(url, this._config.headers, body);
+
       this._config.logger?.debug(`post request to "${url}"`, response);
 
-      if (!isObject(response.body) || response.body.success !== true) {
+      if (
+        !isObject(response.body) ||
+        response.body.success !== true ||
+        response.status < 200 ||
+        response.status >= 400
```

Comment:
I think we can just use response.ok instead of these last two

Reply:
Will do